guides/user/plugins/user-guide.md

```diff
@@ -114,122 +88,145 @@ This guide uses the [file](https://raw.githubusercontent.com/spinnaker-plugin-ex
 ]
 ```
 
+For example:
+
 ```json
 [
   {
-    "id": "examplePluginsRepo",
+    "id": "spinnaker-plugin-examples",
     "url": "https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json"
-  }
+ },
+ {
+     "id": "my-company-internal-plugins",
+     "url": "https://<my-company-internal-github>/<repo-name>/plugins.json"
+ },
+ {
+	  "id": "my-plugins",
+	  "url": "https://github.com/aimeeu/pluginRepository/blob/master/plugins.json"
+ }
 ]
 ```
 
-
-# Configure Spinnaker for the plugin repository
-
-Use `hal plugins repository add REPOSITORY [parameters]` to add the repository file to Spinnaker's configuration:
+Save your `repositories.json` file in a web location that Spinnaker can access. Then you can add a new plugin repository using the `repositories.json` file:
 
 ```
-hal plugins repository add spinnaker-plugin-examples \
-    --url=https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/repositories.json
+hal plugins repository add all-the-plugins \
+    --url=https://raw.githubusercontent.com/aimeeu/all-the-plugins/master/repositories.json
 ```
 
-See the command [reference](/reference/halyard/commands/#hal-plugins-repository) for a complete list of parameters.
+You can also list, edit, and delete repositories. See the command [reference](/reference/halyard/commands/#hal-plugins-repository) for a complete list of parameters.
 
-# Deploy Spinnaker
+Don't forget to `hal deploy apply` to apply your configuration changes.
 
-Use `hal deploy apply` to redeploy Spinnaker with the updated configuration.
+## Add a plugin using Halyard
 
-# Additional plugin repository commands
+After you have added your plugin repository, you can add your plugin to Spinnaker. The Halyard [command](/reference/halyard/commands/#hal-plugins-add) is:
 
-## List configured plugin repositories
-
-```
-hal plugins repository list
+```bash
+hal plugins add <unique-plugin-id> --extensions=<extension-name> \
+--version=<version> --enabled=true
 ```
 
-## Edit a plugin repository
+The plugin distributor should provide you with the `unique-plugin-id`, `extensions`, and `version` values as well as any plugin configuration details. If you have to hunt for these values, you can find `unique-plugin-id` and `version` in the `plugins.json` file, but you have to look at the code to find the value for `extensions`. Search for the `@ExtensionConfiguration` annotation, which defines the extension name.
```

Comment:
`@ExtensionConfiguration` is deprecated (should still be mentioned here). New plugins should use `@PluginConfiguration`. If the annotation is placed on a subclass of `org.pf4j.Plugin`, then the configuration path looks like this:

```
spinnaker:
  extensibility:
     plugins:
       pluginId:
         <unique-key>:
           config:
```

If `@PluginConfiguration` annotates an extension class, then the config path is the same as for `@ExtensionConfiguration`.